src/api/python/cvc5.pxi

```diff
@@ -466,52 +487,98 @@ cdef class Op:
         return indices
 
 cdef class Grammar:
+    """Wrapper class for :cpp:class:`cvc5::api::Grammar`."""
     cdef c_Grammar  cgrammar
     cdef Solver solver
     def __cinit__(self, Solver solver):
         self.solver = solver
         self.cgrammar = c_Grammar()
 
     def addRule(self, Term ntSymbol, Term rule):
+        """
+            Add \p rule to the set of rules corresponding to \p nySymbol.
```

Comment:
Iirc `\p` is Doxygen only -- did you check the generated documentation? Does this do what you would expect? I think you want to use "\``rule``" here.